tools/pipelines/templates/include-vars-node-version.yml

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+variables:
+# The node.js version to be used
+- name: nodeVersion
+  value: '16.16.0'
```

Comment:
@alexvy86 alternatively we could use a default parameter for the `tools/pipelines/templates/include-use-node-version.yml` and not provide it at all in the tasks but this seemed more explicit. Let me know what you think.

Reply:
I think I lean towards having it be the default in the `use` template. It gives us the same flexibility and we don't need to import the `vars` file in every pipeline. I personally dislike variable-definition templates a bit, because they introduce variables in a way that is not obvious at all for someone reading through the pipeline definition (and similar complaint with variable groups).

If we wanted to test _everything_ with a new version of node we'd have to make a change either way, in the `var` template or in the `use` template. If we wanted to test it for a particular pipeline, we could still provide the override in it in both cases.

I'd see the point of the vars file if the variable was used for different purposes in different pipelines, but if everywhere we use it is only to pass it to the use-node task... I think it's better to just put it there directly.